documentation/decisions/adr/2024-04-10-dashboard-tool.md

```diff
@@ -0,0 +1,77 @@
+- ✅ Allows technical users to create ad hoc queries to create graphs - [AWS QuickSight supports creating a variety of visual types](https://docs.aws.amazon.com/quicksight/latest/user/working-with-visual-types.html)
+- ✅ Easy-to-use UI for non-coders - Subjectively, the AWS QuickSight UI was found to be easy to use.
+- ❌ Replicable for users outside of the project - AWS QuickSight is not open source, so its results can only replicated by having access to our AWS account
+- Cost of ownership - A rough estimate puts AWS QuickSight at about ~$300/month for our quantity of users. [Pricing page.](https://aws.amazon.com/quicksight/pricing/)
+- ✅ Ease of deployment - [AWS QuickSight can be deploy via Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/quicksight_account_subscription)
+- ❌ Simple account creation - [AWS QuickSight users must be deployed via Terraform or the AWS console. These users require an associated IAM user to be created.](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/quicksight_user)
+
+### Metabase
+
+> Metabase is an open-source Business Intelligence (BI) tool that enables users to easily query, visualize, and share insights from their data. It offers a user-friendly interface that allows users to create and customize dashboards and visualizations without the need for advanced technical skills. Metabase supports various data sources, including SQL databases like MySQL, PostgreSQL, and MongoDB, as well as cloud services like Google BigQuery and Amazon Redshift. With features such as SQL querying, interactive dashboards, and natural language querying, Metabase empowers users to explore and understand their data in a flexible and intuitive way. Additionally, being open-source, Metabase allows for community contributions and customization, making it a popular choice for organizations seeking a cost-effective and customizable BI solution.
+
+Here's how Metabase evaluates against our criteria:
+
+- ✅ Ability to share public dashboards - [Metabase supports public dashboards](https://www.metabase.com/docs/latest/questions/sharing/public-links)
+- ✅ Ability to show private dashboards to selected users - [Metabase supports access controlled dashboards](https://www.metabase.com/learn/administration/guide-to-sharing-data)
+- ✅ Ability to connect to common data sources (S3, Redshift, Postgres) - [Metabase supports common data sources](https://www.metabase.com/data_sources/)
+- ✅ Allows technical users to create ad hoc queries to create graphs - [Metabase supports creating a variety of visual types](https://www.metabase.com/learn/visualization/)
+- ✅ Easy-to-use UI for non-coders - Subjectively, the Metabase UI was found to be easy to use.
+- ✅ Replicable for users outside of the project - Metabase is open-source and
```

Comment:
Was there more you wanted to add here? 
It seems like metabase will allow for our open source contributors to replicate? That’s great that we can do that with metabase. I’m curious what that looks like. For example, would users outside of the project getting access slow down the querying at all? Do we need to set anything up to protect that from happening? You don’t necessarily have to add it to the ADR, but I’m just curious about it.

Reply:
> Was there more you wanted to add here?

Yes I just finished the sentence 👍🏼 

> I’m curious what that looks like.

For the record, I don't think we should invest in doing this at all. It seems like a non-ideal use of time. Specifically, the time cost to setup / accidental risk of data leakage -versus- benefit is poor.

That said, if we were to support people outside the project replicating our Metabase data, we _**shouldn't**_ do it via giving them direct access to the database. That creates a massive security risk. We _**should**_ do it via giving them a pre-cleaned database dump of our analytics SQL database.